spec/controllers/collections_controller_spec.rb

```diff
@@ -6,6 +6,15 @@
 
   let(:collection) { create(:collection) }
 
+  describe "POST #create" do
+    context "when the header_image_url is invalid" do
+      it "does not error" do
+        post :create, params: { collection: attributes_for(:collection).merge(header_image_url: "This will error.") }
+        expect(response).to have_http_status(:found)
```

Comment:
Please also check for the validation error here (it might be good to also adjust the test name to be clearer for what error is and isn't expected)